class AuthError(Exception):
    pass


class ParamsError(Exception):
    pass
